docs/release-notes.md

```diff
@@ -2,6 +2,28 @@
 title: Release Notes
 ---
 
+## v2.1.0
+
```

Comment:
Let's add a breaking change section, where we highlight the switch of default deduplication strategy to drop

Reply:
Done with 48f3e6f8523ddc751a11262f80185de9cceb7f8f. I will give the new commits another pass once the open PRs are merged.